java-manta-client-unshaded/src/main/java/com/joyent/manta/client/MantaMetadata.java

```diff
@@ -171,8 +172,13 @@ public int hashCode() {
     }
 
     @Override
```

Comment:
Help me understand the change below more. The one on the left seems simple.

Reply:
Refactored ```Object object -> Object o``` for code consistency, and redesigned the structure for ```equals()``` method to ensure consistency since most implementations of ```boolean equals()``` method across the SDK compares of ```nested if & if-else``` conditional statements

Reply:
Also used ```Object.equals()``` sourcing information from [here](https://stackoverflow.com/questions/5038204/apache-commons-equals-hashcode-builder).

Reply:
Null checks aren't required in the ```equals()``` boilerplate implementations across the SDK because IntelliJ does that check itself.
```
[INFO]:expressions whose value is statically proven to be constant, and situations that can lead to nullability contract violations. Variables, method parameters and return values marked as @Nullable or @NotNull are treated as nullable (or not-null, respectively) and used during the analysis to check nullability contracts.
```